File: vogal_ou_consoante.py
vogal = "aeiou"
consoante = "bcdfghjklmnpqrstvwxyz"

print("Verifique se a letra digitada é uma vogal ou consoante")

while True:
    pergunta = input("Digite uma letra: ").lower()

    if pergunta.isalpha():
        if pergunta in vogal:
            print(f"{pergunta} é uma vogal")
            
        elif pergunta in consoante:
            print(f"{pergunta} é uma consoante")
        break
    else:
        print(f"{pergunta} não é uma letra")

File: Vogal_ou_consoante.js
function verifyletra(value){
  var lista = ["b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "q", "r", "s", "t", "v", "w", "x", "y", "z"];
  if(isNaN(value) == false){
    console.log("o valor digitado é um numero");
    console.log("essa função so aceita letras");
  } 
  if(lista.includes(value)){
    console.log(value + ":é uma consoante");
  }else{
    console.log(value + ":é uma vogal");
  }
}

var letra = prompt("digite uma letra: ");
var chamada_func = verifyletra(letra);